### PR TITLE
feat: home page

### PR DESCRIPTION
**Pull request by prajapati-priyanka**
-add home page.

csb link: https://codesandbox.io/s/github/prajapati-priyanka/pine-apparel/tree/home

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  align-items: center;
+  color: var(--lynx-white-color);
+  font-weight: var(--bold-font);
+}
+
+.banner-section .banner-text-overlay h2 {
+  margin-bottom: 1rem;
+}
+
+.banner-section .banner-text-overlay button {
```
Comment (Kevin-Solomon) [inline index.css:68]: Just a suggestion , this will apply the style to every button in ```.banner-section```

**Review by mohammed-sas — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+                <ul>
+                    <li><a href="https://github.com/prajapati-priyanka"><i class="fa fa-github-square"></i></a></li>
+                    <li> <a href="https://www.linkedin.com/in/priyanka-prajapati-853098146/"><i
+                                class="fa fa-linkedin-square"></i></a></li>
+                </ul>
+            </div>
+        </div>
+    </footer>
+</body>
+</html>
```
Comment (Kevin-Solomon) [inline index.html:130]: Just a suggestion ,add new line at the end of the file.

**Review by Kevin-Solomon — APPROVED** [review]

**Review by AnkurChunekar — APPROVED** [review]
priyanka pr
Hey Priyanka,
As you can see in the SS above, icons and login button aren't aligned to right (below 900px width), IMO it would look better and consistent because above 900px width they are aligned to the right.

File: index.css (hunk 1 above)
Comment (prajapati-priyanka) [reply index.css:68]: Actually, there is only one button in the banner section right now, that is why I have written like this. In the future, if I add more buttons then I will change them accordingly.

File: index.html (hunk 2 above)
Comment (prajapati-priyanka) [reply index.html:130]: okay, I will do that. Thanks.

**Pull request merged**